package.json

```
@@ -17,81 +17,60 @@
     "prebuild": "node -e \"process.exit(process.env.PHOVEA_SKIP_TESTS === undefined?1:0)\" || npm run test",
-    "pretest": "npm run compile",
-    "test": "test ! -d tests || karma start",
+    "test": "jest --passWithNoTests",
+    "test:watch": "jest --watch",
     "posttest": "npm run lint",
-    "test:watch": "karma start --autoWatch=true --singleRun=false",
-    "build:dev": "webpack",
-    "build": "webpack --env prod",
-    "start": "webpack-dev-server --inline",
-    "start:hot": "webpack-dev-server --inline --hot",
-    "watch": "webpack --watch",
+    "build": "npm run clean && npm run compile",
+    "copy-assets": "if [ -d src/assets ]; then shx --verbose cp -R src/assets/. dist/assets/; fi",
+    "copy-styles": "if [ -d src/scss ]; then shx --verbose cp -R src/scss/. dist/scss/; fi",
+    "copy": "npm run copy-assets && npm run copy-styles",
     "release:major": "npm version major && npm publish && git push --follow-tags",
     "release:minor": "npm version minor && npm publish && git push --follow-tags",
     "release:patch": "npm version patch && npm publish && git push --follow-tags",
     "predist": "npm run build && npm run docs",
-    "dist": "mkdir dist && cd build && tar cvzf ../dist/tdp_gene.tar.gz *"
+    "dist": "mkdir lib && cd dist && tar cvzf ../lib/tdp_gene.tar.gz *"
   },
   "devDependencies": {
-    "@types/jasmine": "2.5.47",
-    "awesome-typescript-loader": "3.1.2",
-    "css-loader": "0.28.0",
-    "extract-loader": "0.1.0",
-    "extract-text-webpack-plugin": "2.1.0",
-    "file-loader": "0.11.1",
-    "html-loader": "0.4.5",
-    "imports-loader": "0.7.1",
-    "jasmine": "2.5.3",
-    "json-loader": "0.5.4",
-    "karma": "1.5.0",
-    "karma-chrome-launcher": "2.0.0",
-    "karma-firefox-launcher": "1.0.1",
-    "karma-jasmine": "1.1.0",
-    "karma-junit-reporter": "2.0.0",
-    "karma-sourcemap-loader": "0.3.7",
-    "karma-webpack": "2.0.3",
-    "node-sass": "^4.13.1",
-    "null-loader": "0.1.1",
-    "raw-loader": "0.5.1",
-    "sass-loader": "6.0.7",
-    "style-loader": "0.16.1",
-    "tslib": "~1.11.0",
-    "tslint": "5.9.1",
-    "typedoc": "~0.16.10",
-    "typescript": "~3.8.2",
-    "url-loader": "0.5.8",
-    "webpack": "2.3.3",
-    "webpack-dev-server": "2.4.2",
-    "cache-loader": "1.2.0",
-    "ifdef-loader": "2.0.0",
-    "fork-ts-checker-webpack-plugin": "0.4.4",
-    "thread-loader": "1.1.2",
-    "ts-loader": "4.0.1"
+    "@types/jest": "~26.0.5",
+    "@types/d3": "~3.5.36",
+    "identity-obj-proxy": "~3.0.0",
+    "jest": "~26.1.0",
+    "jest-raw-loader": "~1.0.1",
+    "rimraf": "~3.0.2",
+    "shx": "~0.3.2",
+    "ts-jest": "~26.1.3",
+    "tslib": "~2.0.0",
+    "tslint": "~5.20.1",
+    "typedoc": "~0.17.8",
+    "typescript": "~3.9.7"
   },
   "dependencies": {
-    "jStat": "1.8.6",
+    "jstat": "^1.9.3",
     "jquery-ui": "1.12.1",
-    "tdp_core": "github:datavisyn/tdp_core#semver:^9.0.0"
+    "tdp_core": "github:datavisyn/tdp_core#semver:^10.0.1"
```

Did you release tdp_core on npm and pip?
https://www.npmjs.com/package/tdp_core
https://pypi.org/project/tdp-core/

```suggestion
    "tdp_core": "^10.0.1"
```

---

tdp_core is not published anymore.